plugins/woocommerce/tests/e2e-pw/tests/shopper/cart-block-coupons.spec.js

```diff
@@ -137,9 +137,7 @@ test.describe(
 				await page
 					.getByRole( 'button', { name: 'Add a coupon' } )
 					.click();
-				await page
-					.locator( '#wc-block-components-totals-coupon__input-0' )
-					.fill( coupons[ i ].code );
+				await page.getByLabel( 'Enter code' ).fill( coupons[ i ].code );
```

Comment:
nice one, great to see the move away from fragile selectors 👏

Reply:
Thanks, Sam! 😊